atom/browser/api/atom_api_session.cc

```diff
@@ -543,17 +538,18 @@ void Session::DisableNetworkEmulation() {
       network_emulation_token_, network::mojom::NetworkConditions::New());
 }
 
-void WrapVerifyProc(base::Callback<void(const VerifyRequestParams& request,
-                                        base::Callback<void(int)>)> proc,
-                    const VerifyRequestParams& request,
-                    base::OnceCallback<void(int)> cb) {
-  proc.Run(request, base::AdaptCallbackForRepeating(std::move(cb)));
+void WrapVerifyProc(
+    base::OnceCallback<void(const VerifyRequestParams& request,
```

Comment:
Why is this `base::OnceCallback` while the original declaration is `base::RepeatingCallback` for `proc` ?